include/host-build.mk

```diff
@@ -44,6 +44,7 @@ ifneq ($(strip $(HOST_UNPACK)),)
   define Host/Prepare/Default
 	$(HOST_UNPACK)
 	$(Host/Patch)
+	[ ! -d ./src/ ] || $(CP) ./src/* $(HOST_BUILD_DIR)
```

Comment:
somewhat, related to this ; i was wondering if i should put this before the `$(XXX/Patch)` rules;
that would allow for source to be added (in `src`), and then patches to be applied after that, which [to me] sounds wonky/weird;
the kernel seems to do it like that [ though, i'm not entirely sure about it right now ]
the only "disadvantage" of this order, is that the build won't fail if a patch adds a file, and is being overwritten by anything in `src` ; but [personally] I don't see that as a big problem ;